ui/app/src/utils/i18n-legacy.ts

```diff
@@ -41,6 +41,10 @@ const approveDialogMessages = defineMessages({
   introductoryText: {
     id: 'publishDialog.introductoryText',
     defaultMessage: 'Selected files will be published. Hard dependencies are automatically included. Soft dependencies are optional and you may choose which to include.'
+  },
+  submissionCommentValText: {
+    id: 'publishDialog.submissionCommentValText',
```

Comment:
Please change to `publishDialog.submissionCommentFieldLabel` or `publishDialog.submissionCommentFieldPlaceholder` as appropriate.

Reply:
Done